Task1.c

```diff
@@ -0,0 +1,45 @@
+#include <stdio.h>
```

Comment:
```suggestion
#include <stdio.h>

```
Нужно отделять функции от include (и функции между собой) дополнительным переносом строки